source/grid/grid_tools.cc

```diff
@@ -1060,7 +1061,15 @@ namespace GridTools
     const Triangulation<dim, spacedim> &tria = mesh.get_triangulation();
 
     const std::vector< Point<spacedim> > &vertices = tria.get_vertices();
-    const std::vector< bool       > &used     = tria.get_used_vertices();
+
```

Comment:
That actually points out something that the function documentation should mention: That the size of the input array (if non-zero) should equal `tria.n_vertices()`. (And maybe also add that it should not equal `tria.n_used_vertices()`.)